app/helpers/appointment_helper.rb

```diff
@@ -17,13 +17,52 @@ module AppointmentHelper
     'Charity'
   ].freeze
 
+  WHO_IS_YOUR_PENSION_PROVIDER_OPTIONS = [
+    'Abbey life',
+    'Aegon',
+    'Allied Dunbar',
+    'Aviva',
+    'AXA',
+    'Canada Life',
+    'Cannon Lincoln',
+    'Equitable Life',
+    'Legal & General',
+    'Friends Life',
+    'Phoenix Life',
+    'Prudential',
+    'Reassure',
+    'Royal London',
+    'Scottish Widows',
+    'Standard Life',
+    'Sunlife',
+    'Sunlife Financial of Canada',
+    'Zurich'
+  ].freeze
+
   def where_did_you_hear_about_pension_wise_options(appointment)
-    options =
-      WHERE_DID_YOU_HEAR_ABOUT_PENSION_WISE_OPTIONS |
-      [appointment.where_did_you_hear_about_pension_wise]
+    select_options(
+      :where_did_you_hear_about_pension_wise,
+      WHERE_DID_YOU_HEAR_ABOUT_PENSION_WISE_OPTIONS,
+      appointment
+    )
+  end
+
+  def who_is_your_pension_provider_options(appointment)
+    select_options(
+      :who_is_your_pension_provider,
+      WHO_IS_YOUR_PENSION_PROVIDER_OPTIONS,
+      appointment
+    )
+  end
+
+  private
+
+  def select_options(field, options, object)
+    options |=
+      [object.send(field)]
```

Comment:
Do we need to use send. 

Would it be easier to just pass the value in given we only use the field to extract the value from the object (which we also pass in)